<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="programme_data" uuid="ebb16dcb-866e-4336-aa7a-2b8de8a7848c">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:E:\python project\Repair box\db\programme_data.db</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="COUNTIS" uuid="d206b89a-8465-4a76-91ca-68c69995d6b8">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:E:\python project\Repair box\dbmodels\COUNT\COUNTIS.sqlite</jdbc-url>
    </data-source>
  </component>
</project>